src/importexport/capella/tests/tst_capella_io.cpp

```diff
@@ -63,7 +63,7 @@ private slots:
     void capxTest4() { capxReadTest("test4"); }   // wrong enharmonic spelling
     void capxTest5() { capxReadTest("test5"); }
     void capxTest6() { capxReadTest("test6"); }
-    void capxTest7() { capxReadTest("test7"); }   // double bar missing (auto-generated by Capella programs)
+    //   void capxTest7() { capxReadTest("test7"); } // wrong cap xml ( element double close (<head pitch="E5"/> </head>) )  // double bar missing (auto-generated by Capella programs)
```

Comment:
Test commented out regardless the test file having been changed?

Reply:
I planned to fix it... but then I abandoned this solution as part of this task. The content is not changed there, probably just different compression :) I will return the original file